Cargo.toml

```diff
@@ -0,0 +1,26 @@
+
+
+[workspace]
+# We want to use v2 of the cargo dependency resolver.
+resolver = "2"
+# Our workspace members include the packages in the contracts directory.
+members = ["contracts/voting", "contracts/raffle"]
+
+
```

Comment:
So, just to confirm, being this profiles in the workspace, they have effect in all crates members right ?

Reply:
You mean the profiles? Yeah they should be set at workspace level. Setting them on a specific crate has no effect.